cypress/integration/participantView.spec.js

```diff
@@ -37,15 +37,15 @@ describe("Participant View", () => {
     cy.get('div.MuiSelect-selectMenu').should('not.have.class', 'Mui-disabled');
     cy.get('div.MuiSelect-selectMenu').click();
     cy.get('ul.MuiMenu-list[role=listbox]').should('be.visible');
-    cy.get('ul.MuiMenu-list[role=listbox] > li').should('have.length', 6);
+    cy.get('ul.MuiMenu-list[role=listbox] > li').should('have.length', 7);
   });
 
   it("Visits Participant View as a MoH user", () => {
     cy.kcNavAs('ministry_of_health', 'participant-view');
     cy.get('div.MuiSelect-selectMenu').should('not.have.class', 'Mui-disabled');
     cy.get('div.MuiSelect-selectMenu').click();
     cy.get('ul.MuiMenu-list[role=listbox]').should('be.visible');
-    cy.get('ul.MuiMenu-list[role=listbox] > li').should('have.length', 6);
+    cy.get('ul.MuiMenu-list[role=listbox] > li').should('have.length', 7);
```

Comment:
Question: if we have None as an option, should "Preferred Location" still be available as a catch-all? Do they serve different purposes?